cli/cmd/cluster.go

```diff
@@ -24,9 +24,14 @@ func createCobraCommand(use string, short string, cmd *command) *cobra.Command {
 	if cmd == nil {
 		return &cobra.Command{Use: use, Short: short}
 	}
+
+	// creating a clusterData object to call getHost
+	var emptyClusterData *ClusterData
+	hostName := emptyClusterData.getHost()
+
 	return &cobra.Command{
 		Use:   use,
-		Short: short,
+		Short: fmt.Sprintf( short, hostName, hostName),
```

Comment:
```suggestion
		Short: fmt.Sprintf(short, hostName, hostName),
```
why we have 2 `hostName` here?

Reply:
This is changed. Not formatting this field not.